### PR TITLE
Document packit dist-git init

### PR DESCRIPTION
**Pull request by lbarcziova**
Related to packit/packit#2225

### CODE REVIEW

File: docs/cli/dist-git/init.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    Usage: packit dist-git init [OPTIONS] [PATH_OR_URL]
+    
+      Create the Packit dist-git configuration for Fedora release syncing.
+    
+      This command adds `.packit.yaml` file to the dist-git repository either:
+    
+      1. specified by path (defaults to current working directory)
+    
+      2. specified by URL (`https://src.fedoraproject.org/rpms/<package>`) -
+      clones the repository and adds the config in there. Ideally use this with
```
Comment (majamassarini) [inline docs/cli/dist-git/init.md:23]:
```suggestion
      clones the repository and adds the config there. Ideally use this with
```
This sounds more to me, but I am not sure at all

**Review by majamassarini — APPROVED** [review]
Sounds good to me. Thanks.

File: docs/cli/dist-git/init.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+---
+
+# `packit dist-git init`
+
+Create a Packit's dist-git configuration for release syncing to Fedora.
+
+
+## Help
+    
+    $ packit dist-git init
```
Comment (mfocko) [inline docs/cli/dist-git/init.md:13]: I guess I could've created a card to automate this thing somehow, I think that I had to fix some inconsistencies a long time ago…

File: docs/configuration/downstream/bodhi_update.md [hunk 3]
```diff
@@ -33,6 +33,13 @@ for creating your Packit configuration in dist-git repository.
 
 :::
 
+:::tip Automate the setup
+
+You can also use [packit dist-git init](/docs/cli/dist-git/init.md) CLI command to create your
+Packit dist-git configuration.
```
Comment (mfocko) [inline docs/configuration/downstream/bodhi_update.md:39]:
```suggestion
initial Packit dist-git configuration.
```

or

```suggestion
Packit dist-git configuration that you can customize to your needs.
```

**Review by mfocko — APPROVED** [review]
LGTM

**Review by TomasTomecek — APPROVED** [review]

File: docs/cli/dist-git/init.md (hunk 2 above)
Comment (lbarcziova) [reply docs/cli/dist-git/init.md:13]: good idea :+1:

**Comment by lbarcziova** [comment]
regate

File: docs/cli/dist-git/init.md (hunk 2 above)
Comment (lbarcziova) [reply docs/cli/dist-git/init.md:13]: created #825

**Pull request merged**